gui/index2.html

```diff
@@ -4,8 +4,15 @@
 	<title>MiniBot GUI</title>
 	<!-- IMPORTS -->
 	<link rel="stylesheet" href="./static/css/gui_update.css"/>
-
+	<link rel="stylesheet" href="./static/css/vendor/prettify/prettify.css">
```

Comment:
Minor: Consider writing an `import.js` file that imports all necessary stylesheets and scripts into the GUI index so that it's not cluttered with a bunch of file imports at the top.